ui/ducks/send/send.js

```diff
@@ -44,6 +44,7 @@ import {
   hideLoadingIndication,
   showConfTxPage,
   showLoadingIndication,
+  updateTokenType,
```

Comment:
So while trying to test this I re-found an issue similar to one I had been having while trying to tack[ a PR on top of the broad send-slice refactor](https://github.com/MetaMask/metamask-extension/pull/11335). 
Here it is, so entering the send flow the form status is immediately set to valid:
<img width="522" alt="Screen Shot 2021-06-23 at 12 05 52 PM" src="https://user-images.githubusercontent.com/34557516/123139474-dd7b8f80-d41b-11eb-8992-68eaa937559c.png">

once, however, I click the dropdown to select another token, the form becomes invalid:
<img width="514" alt="Screen Shot 2021-06-23 at 12 06 22 PM" src="https://user-images.githubusercontent.com/34557516/123139564-fa17c780-d41b-11eb-8f2e-c455e1aab579.png">

That is expected, but then I cannot get the form status to be valid again: 
<img width="515" alt="Screen Shot 2021-06-23 at 12 06 29 PM" src="https://user-images.githubusercontent.com/34557516/123139636-0ac83d80-d41c-11eb-9db5-9b6f8c8f553c.png">

even if I go back and select Eth as the send asset:
<img width="472" alt="Screen Shot 2021-06-23 at 12 11 24 PM" src="https://user-images.githubusercontent.com/34557516/123139847-406d2680-d41c-11eb-91fa-2a6de1043900.png">

I believe it has something to do with sequencing of actions and the gas loading thunk. The last call to validateSendState sets us to invalid [here](https://github.com/MetaMask/metamask-extension/blob/f7bb6c39ebf8c59fdc2abc95bad151bcaf8a451e/ui/ducks/send/send.js#L846).

Reply:
can you try https://github.com/MetaMask/metamask-extension/pull/10965/commits/3d8ca2d7f98528957fc847da06547720f9070735 out for a spin? see if the issue persists?

Reply:
@brad-decker That commit seems to have fixed the issue, 🚀